mod-di-converter-storage-server/src/main/resources/templates/db_scripts/associations-migration/init_wrappers.sql

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
The trigger updates the wrapper id columns on the table when the jsonb column is updated which is what we want. By the time the profile migration service runs, the schema is already upgraded for wrappers. That is why the triggers were not disabled. Do you see an issue?

There isn't any usage of association_view since we needed to old version and the schema was already upgraded. That is why I opted to connect the wrappers without it. It is a gap in migration use cases since we want to do this migration before the schema is upgraded.

Reply:
@okolawole-ebsco I just mean - will be there any issues if we run the current code/migration implementation as initial migration to the profile wrappers feature? I mean not from the previous profile_wrappers version, just from the version where the module doesn't know smth about profile_wrappers. In this case - how it will be upgraded to the current implementation, will there be any issues? Or maybe I missed something?

Reply:
The code in this branch should successfully upgrade a Orchid version to Poppy. The new versions of the association _views etc are created after this script is executed. This script is executed by the ProfileMigrationService and then it invokes another script to create the needed views and functions. Hopefully I answered your question. Reach out to me on slack so that we can chat live. Thanks!